src/elements/content-sidebar/activity-feed/comment/Comment.scss

```diff
@@ -41,3 +41,9 @@
 .bcs-Comment-unresolveComment path {
     fill: $bdl-gray;
 }
+
+.bcs-ActivityThread {
```

Comment:
Components should be encapsulated and not aware of how/where they're being used. Can we move this to the `ActivityThread` stylesheet?